Webdev - Add Description Metadata (#13088)
hashicorp/vagrant: website/content/vagrant-cloud/support.mdx

---
layout: vagrant-cloud
page_title: Vagrant Cloud Support

> description: "Vagrant Cloud users can find support through our mailing lists"

Reviewer: ```suggestion
description: "Find support for Vagrant Cloud through our email or community channels."
```

🤔 I think this could be maybe better described like such? Feel free to adapt it. The gist is that we have other support than just mailing lists, especially for people that pay us.
